code/game/objects/items/circuitboards/computer_circuitboards.dm

```diff
@@ -241,7 +236,7 @@
 	greyscale_colors = CIRCUIT_COLOR_GENERIC
 	build_path = /obj/machinery/computer/arcade/orion_trail
 
-/obj/item/circuitboard/computer/holodeck// Not going to let people get this, but it's just here for future
+/obj/item/circuitboard/computer/holodeck // Not going to let people get this, but it's just here for future
```

Comment:
probably isn't needed either, a lot of these are just like coder thoughts but not actually needed for anyone to work with the code (aka REMOVE THEM)